BASE/src/Microsoft.ApplicationInsights/Microsoft.ApplicationInsights.csproj

```diff
@@ -35,4 +35,4 @@
     <AdditionalFiles Include="$(PublicApiRoot)\$(AssemblyName).dll\$(TargetFramework)\PublicAPI.Unshipped.txt" />
   </ItemGroup>
   
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
nothing changed?

Reply:
Github complains if a file is missing a new-line at the very end. That's the red circle on line 38.